test/python/WMCore_t/Services_t/DBS_t/DBSUtils_t.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+"""
+_DBSUtils_t_
+
+Unit test for the DBSUtils module
+"""
+
+import time
+import unittest
+
+from WMCore.Services.DBS.DBSUtils import urlParams
+from WMCore.Services.DBS.DBS3Reader import DBS3Reader
+
+
+class DBSUtilsTest(unittest.TestCase):
+
+    def testUrlParams(self):
+        """
+        urlParams should return dictionary of URL parameters
+        """
+        url = 'http://a.b.com?d=1&f=bla'
+        results = urlParams(url)
+        self.assertTrue(results.get('d'), 1)
+        self.assertTrue(results.get('f'), 'bla')
```

Comment:
Maybe testing `results` with:
```
self.assertCountEqual(results, {'d': ['1'], 'f': ['bla']})
```
would be cleaner and better?

Can you please also add a test case for an url with multiple values for the same query string (e.g. d=1&d=2&f=bla)?

Reply:
done